Dockerfile

```
@@ -1,40 +1,43 @@
-FROM rust:latest AS chef
+FROM --platform=$BUILDPLATFORM rust:1.73.0-alpine3.18 AS chef
 # We only pay the installation cost once,
 # it will be cached from the second build onwards
+RUN apk add --update alpine-sdk musl-dev g++ make libcrypto3 libressl-dev git upx perl build-base && rm -rf /var/cache/apk/*
```

```suggestion
RUN apk add --no-cache alpine-sdk musl-dev g++ make libcrypto3 libressl-dev git upx perl build-base
```

Also, I would like to suggest making a minor change like the new alpine image have recently add the `--no-cache` option which does some things automatically like removes the `/var/cache/apk/•` and updates the repository automatically. 🙂

---

Ah yes i'm too used to the old way